PF/Assignment1/Survey.java

```diff
@@ -0,0 +1,127 @@
+//To conduct a survey using open end methodology
+import java.util.Scanner;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Survey {
+	private static Scanner scan;
+
+	public static void main(String args[]) {
+		try {
+			scan = new Scanner(System.in);
+			BufferedReader br = new BufferedReader(new InputStreamReader(
+					System.in));
+			PrintQuestion object1 = new PrintQuestion();
+			Answer object2 = new Answer();
+			int ch, count = 0, i, result;
+			double totalA = 0.0, totalB = 0.0, totalC = 0.0, totalD = 0.0;
+			char choice;
+			String[] multiChoice = new String[10];
+			String answer = "";
+			String a = ""; // menu drive for user
+			do {
+				System.out.println("Menu");
+				System.out.println("1.Take survey");
+				System.out.println("2.View Statistics");
+				System.out.println("3.View people and answers");
+				System.out.println("Enter your choice");
+				ch = scan.nextInt();
+
+				switch (ch) {
+				case 1:
+					count = count + 1; // To ask survey questions from user
+					System.out.println(object1.singleSelect());
+					result = scan.nextInt();
+					if (result < 0 || result > 5) { // to check for correct
```

Comment:
ACTION: can we not use do while loop instead of if else here?. in do we will simply set the single answer object2.singleAnsSet(result, count); and in while we will apply valid input check. so if user answer is valid (1-4) then it  will set value in do and will not execute while and if it is answer is not valid then it will execute while loop to take valid input from user.